Automattic/wp-calypso
# Update/new performance UI

Closes https://github.com/Automattic/dotcom-forge/issues/9142

## Proposed Changes

* Adds the new performance circular design and display it's details when selected
* Create a circular progress to show performance score
* Adds a `showV2` property that determine if the new design should be shown or not

![image](https://github.com/user-attachments/assets/9997f09e-77c4-449d-8eb3-a28a4f619bce)


## Why are these changes being made?


* This change was added as an update to the core web vitals section's design

## Testing Instructions



* Set `wpcom_performance_url` in this file `client/hosting/performance/site-performance.tsx`
* Go to `/site-performance/atomic_site` 
* Verify the new design is shown 
* Verify `http://calypso.localhost:3000/speed-test-tool` shows version one of the design

## Pre-merge Checklist
